datafusion/sqllogictest/test_files/aggregate_skip_partial.slt

```diff
@@ -267,6 +282,20 @@ FROM aggregate_test_100_null GROUP BY c2 ORDER BY c2;
 4 11 14
 5 8 7
 
+# Test avg for tinyint / float
```

Comment:
just wondering why the test is only for tinyint / floats?

Reply:
The idea was that `AVG` the accumulator is already tested elsewhere -- this test is only to exercise the partial aggregate skipping logic